code/components/extra-natives-five/src/NuiAudioSink.cpp

```diff
@@ -2038,6 +2038,34 @@ static InitFunction initFunction([]()
 		{
 			g_submixIds.erase(player);
 		}
+	});
+
+	fx::ScriptEngine::RegisterNativeHandler("SET_AUDIO_SUBMIX_OUTPUT_VOLUMES", [](fx::ScriptContext& ctx)
+	{
+		int sourceIdx = ctx.GetArgument<int>(0);
+		uint32_t effectIdx = ctx.GetArgument<uint32_t>(1);
+		float leftVolume = ctx.GetArgument<float>(2);
+		float rightVolume = ctx.GetArgument<float>(3);
+		float rearLeftVolume = ctx.GetArgument<float>(4);
+		float rearRightVolume = ctx.GetArgument<float>(5);
+		float centerVolume = ctx.GetArgument<float>(6);

```

Comment:
Have you confirmed the order in this case?

Reply:
I made some more tests with a 5.1 audio cable.
1 is left and seems to range from 0.0 to 1.0
2 is right and seems to range from 0.0 to 1.0
3 is surround left and seems to range from -1.0 to 1.0
4 is surround right and seems to range from -1.0 to 1.0
5 is unknown but not used in voice chat
6 is unknown but not used in voice chat

5 or 6 is center, other one must be lfe but cannot say for sure...

The game is not using center speaker (except for some menu sound and some minor stuff) and lfe also seems to not work according to reports online.

I made a video you can see the volume changes in the audio cable manager.
https://www.youtube.com/watch?v=AVf5NKslZR8